src/zenml/zen_server/routers/projects_endpoints.py

```diff
@@ -511,6 +511,48 @@ def list_project_pipelines(
     )
 
 
+@router.get(
+    "/{project_name_or_id}" + PIPELINES,
```

Comment:
Is this really the correct endpoint to list runs?

Reply:
good catch